Builds/Test.py

```diff
@@ -265,6 +269,11 @@ def main():
                 os.remove('.ninja_deps')
                 os.remove('.ninja_log')
 
+    if all_failed:
+        if len(to_build) > 1:
+            print()
+            print('FAILED:', *(':'.join(f) for f in all_failed))
+        exit(1)
```

Comment:
Nit: We probably want `sys.exit`, not `exit`. See http://stackoverflow.com/questions/6501121/the-difference-between-exit-and-sys-exit-in-python